test/unit/adapters/EOFeedAdapter.t.sol

```diff
@@ -86,6 +86,12 @@ contract EOFeedAdapterTest is EOFeedAdapterTestUninitialized {
         assertEq(_feedAdapter.latestAnswer(), int256(RATE1));
     }
 
+    function test_LatestAnswerWithDecimals() public {
+        EOFeedAdapter feedAdapter = EOFeedAdapter(Upgrades.deployTransparentProxy("EOFeedAdapter.sol", proxyAdmin, ""));
```

Comment:
Why are we deploying a new feed here?
Why not use "_feedAdapter" like in other tests?

Reply:
because deploying it is the only way to control the parameters.. we could right a harness to do this, might be a good idea, is that your direction?